### PR TITLE
kappa-*, KaSim: update urls to opam cache

### PR DESCRIPTION
**Pull request by mseri**
The git archives have changed checksum since some time and they
keep failing in the CI. See also https://github.com/Kappa-Dev/KappaTools/issues/636

Signed-off-by: Marcello Seri <marcello.seri@gmail.com>

### CODE REVIEW

File: packages/KaSim/KaSim.4.0.0/opam [hunk 1]
```diff
@@ -34,11 +34,9 @@ build: [
...
 install: [ [ make "install-lib" ] ]
-remove: [ [ "ocamlfind" "remove" "KappaLib" ] ]
 
-synopsis: "Software suite for the Kappa language."
-flags: light-uninstall
+synopsis: "Software suite for the Kappa language"
 url {
-  src: "https://github.com/Kappa-Dev/KaSim/archive/v4.0.tar.gz"
+  src: "https://opam.ocaml.org/cache/md5/89/894a517c7991f42100d2e957efc6cfa2"
   checksum: "md5=894a517c7991f42100d2e957efc6cfa2"
```
Comment (kit-ty-kate) [inline packages/KaSim/KaSim.4.0.0/opam:41]:
```suggestion
  src: "https://github.com/Kappa-Dev/KappaTools/archive/v4.0.tar.gz"
  checksum: "md5=2f9ee76a8aa8bf2ad6bbb37071759e59"
```
`diff -r` does not show any difference between the two archives apart from their top directory name

File: packages/kappa-binaries/kappa-binaries.4.1.0/opam [hunk 2]
```diff
@@ -27,7 +27,7 @@ build: [
 synopsis: "Command line interfaces of the Kappa tool suite"
 
 url {
-  src: "https://github.com/Kappa-Dev/KappaTools/archive/v4.1.tar.gz"
+  src: "https://opam.ocaml.org/cache/sha256/f2/f24a38f73a88f1b3d8137aa083bad8d76c679c4b6c59ec29e1567ad95bf1df13"
   checksum: [
     "sha256=f24a38f73a88f1b3d8137aa083bad8d76c679c4b6c59ec29e1567ad95bf1df13"
     "sha512=6391d347817a8afc52f9ae121a5778c065cfb21800d462642a962e4c371dc3348e0e5d1c43cf2ab55e800538cff2e384f423343813d446f7353716ac7475df0d"
```
Comment (kit-ty-kate) [inline packages/kappa-binaries/kappa-binaries.4.1.0/opam:33]:
```suggestion
  src: "https://github.com/Kappa-Dev/KappaTools/archive/v4.1.tar.gz"
  checksum: [
    "sha256=8ca79f1917e3b5b0e3606547d8be374321f8eb68dfbe62e805339ac82bc89fd1"
    "sha512=7c944f27fd2b6752e48e2731b9217954b1f1624b30a5439ab00881d1def5b75aafb044051bdce858a1c766e407a888afeded831705b17fffc35bda02aae45eac"
```

The only difference is (using diff -ru):
```
diff -ru a/KappaTools-4.1/man/version.tex.skel b/KappaTools-4.1/man/version.tex.skel
--- a/KappaTools-4.1/man/version.tex.skel	2020-09-15 15:51:46.000000000 +0100
+++ b/KappaTools-4.1/man/version.tex.skel	2020-09-15 15:51:46.000000000 +0100
@@ -1 +1 @@
-\def\version{HEAD -> master, tag: v4.1}
+\def\version{tag: v4.1}
```

Comment (mseri) [reply packages/kappa-binaries/kappa-binaries.4.1.0/opam:33]: `diff -ru`... this is very useful to know! Thanks

**Comment by mseri** [comment]
I am happy to just change the checksums, but when github regenerates the artifacts it may fail again. I was hoping the project would upload stable artifacts at a certain points, and temporarily we moved the packages to the cache

**Comment by mseri** [comment]
Thanks for the change (and sorry for not making it before)

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**